tests/baselines/reference/importDeclWithDeclareModifier.js

```diff
@@ -15,3 +15,16 @@ Object.defineProperty(exports, "__esModule", { value: true });
 exports.a = void 0;
 exports.a = x.c;
 var b;
+
+
+!!!! File importDeclWithDeclareModifier.js differs from original emit in noCheck emit
+//// [importDeclWithDeclareModifier.js]
+===================================================================
+--- Expected	The full check baseline
++++ Actual	with noCheck set
+@@ -1,5 +1,3 @@
+ "use strict";
+ Object.defineProperty(exports, "__esModule", { value: true });
+-exports.a = void 0;
+-exports.a = x.c;

```

Comment:
I think this is a case where the `noCheck` output is an improvement. As a perf optimization, I made reference marking with an `Unspecified` hint skip all `Ambient` nodes - that causes this change. The `export declare import` statement in this file usually gets transformed into the output js, even though it has a `declare` modifier. That's likely because the `declare` modifier on an import is an error, so the alias marking is done unconditionally. In any case, since the input has a syntax/grammar type error, I think it's fine for the output to diverge a bit like this. I could bring it in-line (either by changing normal behavior to elide this, or `noCheck` to retain it), but I'm not sure it's worth the effort, given the error.